db/schema.rb

```diff
@@ -645,7 +645,7 @@
     t.boolean "sparsity_uplift"
     t.boolean "pupil_premium_uplift"
     t.uuid "delivery_partner_id"
-    t.index ["cpd_lead_provider_id", "participant_profile_id", "declaration_type", "course_identifier", "state"], name: "unique_declaration_index", unique: true, where: "((state)::text = ANY (ARRAY[('submitted'::character varying)::text, ('eligible'::character varying)::text, ('payable'::character varying)::text, ('paid'::character varying)::text, ('clawed_back'::character varying)::text]))"
+    t.index ["cpd_lead_provider_id", "participant_profile_id", "declaration_type", "course_identifier", "state"], name: "unique_declaration_index", unique: true, where: "((state)::text = ANY ((ARRAY['submitted'::character varying, 'eligible'::character varying, 'payable'::character varying, 'paid'::character varying, 'clawed_back'::character varying])::text[]))"
```

Comment:
could we avoid committing these changes please?

Reply:
@cwrw why am I getting these changes after running rails db:migrate if I have the standard setup?

Reply:
Anyway I have reverted the change. Not sure if my version of postgres might be the cause.

Reply:
I think its our different PG versions (I get this too) as you mentioned, but others on the team don't

Reply:
Yes I get this too.